firmware/lucidgloves-firmware/AdvancedConfig.h

```diff
@@ -27,24 +34,35 @@
 #endif
 
 
+#define MULTIPLEXER_DELAY   5 //How many microseconds should be delayed between multiplexer reads
+
+
 //ANALOG_MAX OVERRIDE:
-//uncomment and set as needed (only touch if you know what you are doing)
-//#define ANALOG_MAX 4095 
+//#define ANALOG_MAX 4095 //uncomment and set as needed
 
 #ifndef ANALOG_MAX
-#error "This board doesn't have an auto ANALOG_MAX assignment, please set it manually by uncommenting ANALOG_MAX OVERRIDE!"
+#error "This board doesn't have an auto ANALOG_MAX assignment, please set it manually by uncommenting the line below ANALOG_MAX OVERRIDE in AdvancedConfig.h"
 #endif
 
 //Filtering and clamping analog inputs
 #define CLAMP_ANALOG_MAP true //clamp the mapped analog values from 0 to ANALOG_MAX
 
 // Enable and set min and max to match your sensor's expected raw value range
 // This discards any spurious values outside of the useful range
-#define CLAMP_FLEXION false  //clamp the raw flexion values
+#define CLAMP_SENSORS false  //clamp the raw sensor values
```

Comment:
did you mean to change this?

Reply:
As mentioned before I'm not entirely sure what happenend to AdvancedConfig.h, I thought I used the newest version of proto5, but I can try to change it to match main as the branching somehow does not let me to push to proto5